Svenbjorn/whatson
# Feature/tour menu

TO TEST:

Go to a tour in whatson.
Look at the grey local navigation bar
- EXPECT: It looks like the navigation bar in the sendistovan design:
https://sendistovan.invisionapp.com/console/share/MDFJWAUZY7G/652738057/inspect
- EXPECT: It has been populated with menu points corresponding to the section headers on the page
- Click on menu items
- EXPECT: sections will scroll into view
